airbyte-webapp/src/components/connection/ConnectionForm/FormFieldWrapper.module.scss

```diff
@@ -3,7 +3,7 @@
 .leftFieldCol {
   flex: 1;
   max-width: 640px;
-  padding-right: 30px;
+  padding-right: variables.$spacing-xl;
```

Comment:
This may have precedeed the use of `gap`, which can be done in the `FlexContainer`

Reply:
Duly noted! Fixed! Thanks!